src/PyDynamic/misc/tools.py

```diff
@@ -329,5 +335,56 @@ def progress_bar(
         COUNT=count + 1,
         COUNTMAX=count_max,
     )
+    if fout is not None:
+        fout.write(progressString)
+    else:
+        print(progressString)
+
+def is_vector(ndarray: np.ndarray) -> bool:
+    """Check if a np.ndarray is a vector, i.e. it expands over one dimension only
```

Comment:
This can/should be clarified

```suggestion
    """Check if a np.ndarray is a vector, i.e. it expands over one dimension only, that is of shape (n,)
```

Reply:
Resolved by c0f93be6.